lib/fog/vcloud_director/requests/compute/instantiate_vapp_template.rb

```diff
@@ -41,20 +41,25 @@ def populate_uris(options = {})
           options[:vdc_uri] =  vdc_end_point(options[:vdc_id])
           options[:network_uri] = network_end_point(options[:network_id]) if options[:network_id]
           options[:template_uri] = vapp_template_end_point(options[:template_id]) || raise("template_id option is required")
+          options[:endpoint] = end_point
           options
         end
 
         def generate_instantiate_vapp_template_request(options ={})
-          #overriding some params so they work with new standardised generator
-          options[:InstantiationParams] =
-          {
-            :NetworkConfig =>
-              [{
-                :networkName => options[:network_name],
-                :networkHref => options[:network_uri],
-                # :fenceMode => "bridged"
-              }]
-          } unless options[:InstantiationParams]
+          # TODO: this is old format for customizing vapp networks, please use :vapp_networks instead to let
```

Comment:
Please open an issue to remove the legacy format as it should be removed in the future.